pep-0501.txt

```diff
@@ -571,12 +723,19 @@ References
 
 * :pep:`498`: Literal string formatting
 
+* :pep:`675`: Arbitrary Literal String Type
+
+* :pep:`701`: Syntactic formalization of f-strings
+
 * `FormattableString and C# native string interpolation
   <https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated>`_
 
 * `IFormattable interface in C# (see remarks for globalization notes)
   <https://docs.microsoft.com/en-us/dotnet/api/system.iformattable>`_
 
+* `TemplateLiterals in Javascript
+  <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals>`_
+
 * `Running external commands in Julia
   <https://docs.julialang.org/en/v1/manual/running-external-programs/>`_
 
```

Comment:
I can't quite reach it with a comment, but since @ncoghlan and @nhumrich are the only substantive contributors to the current PEP, I suggest you agree to revise the copyright statement following our modern requirements:

```rst
This document is placed in the public domain or under the
CC0-1.0-Universal license, whichever is more permissive.
```